src/components/combo-box/combo-box.ts

```diff
@@ -152,6 +152,7 @@ class BXComboBox extends BXDropdown {
         .value=${filterInputValue}
         role="combobox"
         aria-label="${inputLabel}"
+        aria-labelledby="assistiveStatus"
```

Comment:
Sounds that `aria-label` above is no longer used...?

Reply:
You're right, I'll remove it

Reply:
Another question. `<div id="assistiveStatus">` has `aria-live="assertive"`, which means screen reader should read it whenever the content changes. Did you mean it was not the case?

Reply:
The `aria-live` content was being read out normally using Mac Voiceover, but without `aria-labelledby` the content wasn't being read out using JAWS for some reason.

Reply:
I see, would you try taking deeper look at how JAWS works with `aria-live`? You can try https://www.w3.org/TR/wai-aria-practices/examples/alert/alert.html for example. Some ARIA site suggests that `role="alert"` or `role="status"` in place of `aria-live`: https://www.w3.org/TR/WCAG20-TECHS/ARIA19.html#ARIA19-ex1 https://www.w3.org/WAI/WCAG21/working-examples/aria-role-status-searchresults/ - Would you try those, too? Thanks!

Reply:
The current `<div id="assistiveStatus">` currently has `role="status"` along with `aria-live`. Removing `aria-live` still makes it work, but `aria-labelledby` is still needed for it to be read out.

Reply:
My apologies I was not clear enough; The links above has code examples. Would you try those code examples with JAWS to see how `aria-live`, `role="alert"` and `role="status"` work? Thanks!

Reply:
From those examples, JAWS reads out the message for a split second then returns to what it was previously reading. It could be because I'm running JAWS in a Virtual Machine that it behaves that way, but I'm not sure. The Dropdown/Combobox changes worked alright on the VM, but I usually asked Ram to test out changes to confirm they worked in his machine.

Reply:
I see, thanks. The next steps would be:

* Work with India team to understand what they seem from W3 ARIA examples (above)
* If the result is what we expect (JAWS read out stuffs as soon as we put the content in), reduce our code or enhance W3 code to see what the fundamental differences are

The bottom line: `aria-live`, `role="alert"` or `role="status"` should work without being referred from our combo box input.